.pre-commit-config.yaml

```diff
@@ -8,9 +8,10 @@
         args: [--quiet]
     -   id: flow
         name: flow
-        entry: shared/node_modules/.bin/flow
+        entry: sh
         language: system
         files: \.js$
+        args: [-c, cd shared && node_modules/.bin/flow]
```

Comment:
flow doesn't need (and is confused by) the changed files passed in, use the old way